Makefile

```diff
@@ -95,7 +95,7 @@ deletenamespaces:
 
 # Install the postgrescluster CRD
 install:
-	$(PGO_KUBE_CLIENT) apply -k ./config/crd
+	$(PGO_KUBE_CLIENT) apply --server-side=true --force-conflicts -k ./config/crd
```

Comment:
:pick: 
Not a blocker, but perhaps this should go in a separate commit?

Reply:
Removed, I think we have another PR where this might fit more naturally.